lib/util.sh

```diff
@@ -68,12 +68,36 @@ increase_val() {
 chk_conf_val() {
     declare -r val2chk=$1
     declare -r def_val=$2
-    eval val2set=\$$val2chk
+    val2set=${!val2chk}
     if [ -z "$val2set" ]; then
         log "ATTENTION: $val2chk not set in sysconfig file."
         log "Setting default value '$def_val' instead"
-	echo -n "$def_val"
+        echo -n "$def_val"
     else
-	echo -n "$val2set"
+        echo -n "$val2set"
+    fi
+}
+
+# check, if a value is defined in the (sourced) sysconfig file
+# if not, log a warning and leave the current system value unchanged
+# otherwise set the value from sysconfig file as the new system value
+chk_and_set_conf_val() {
```

Comment:
I liked that we are still logging what decision was made by sapconf. It helps during debugging.